pkg/v1/tkg/test/tkgctl/aws_cc/aws_cc_antrea_test.go

```diff
@@ -19,11 +19,7 @@ var _ = Describe("Functional tests for aws - Antrea", func() {
 			ArtifactsFolder: artifactsFolder,
 			Cni:             "antrea",
 			Plan:            "devcc",
-			OtherConfigs: map[string]string{
-				"AWS_VPC_ID":            "vpc-0f6da14d1b6f94a73",
```

Comment:
curious: did this not work?

Reply:
Because we need this for MC and WC both, we moved it to generic config file `pkg/v1/tkg/test/config/aws.yaml`